python/instrumentation/openinference-instrumentation-openai/src/openinference/instrumentation/openai/package.py

```diff
@@ -0,0 +1,2 @@
+_instruments = ("openai >= 1.0.0",)
+_supports_metrics = False
```

Comment:
Are the contents of this file OTEL-related?

Reply:
I copied it from otel. See [here](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/14d7779366ec99800f1e9a1f7137c01bc1aff617/instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/package.py#L16) for example.